projects/planner/src/app/plan/diveoptions/diveoptions.component.spec.ts

```diff
@@ -52,5 +52,40 @@ describe('Dive options component', () => {
         fixture.detectChanges();
         component.isComplex = false;
         expect(spy).toHaveBeenCalledWith(false);
-    }));
+    }
+    ));
+
+    describe('Option methods', () => {
+        let schedules: DiveSchedules;
+        let fb: NonNullableFormBuilder;
+
+        beforeEach(inject(
```

Comment:
beforeEach not needed see first 50 lines.
And put everything into one describe.